### DISCUSSION TITLE
Lots of annoying details

### DISCUSSION THREAD

**Discussion by krzysiobal** [TeamNewPipe/NewPipe#10511] [Questions]
Few days ago official youtube application stoppded supporting Android 5.0.1 devices, no updates are available, so I started to looking for some alternatives. i found your project, though it started be nightmare
1) trying to download: google >> newpipe >> found your webstire (newpipe.net)
When I go there, I click "`Get NewPipe` button, then I see a big red icon with triangle, I click the `New Pipe` red button and I am redirected to github.com. No way to download the app (when doing the same on PC web browser, download windows for .apk file appears)

2)So I had to search somewhere else for .apk file, got it from apkpure.com, got the newest version that was available at that day (0.25.1), after instlaling it and trying to launch any video, I got Guru Medidation error (no video streams found). One day later when I searched for updates, I found 0.25.2 was released, with problem solved

3) Why when playing, video stops every 10-20 seconds for few moments and the spinning icon appears (buffering).
I am playing 480p quality, 80 Mbit internet, never have that slowdowns on PC or original youtube app.

4) Why when you rotate the screen, black screen appears for about 3-5 seconds and then video resumes play? Original youtube app was instant

5) Why when you start video, press `back button` two times in order for searching for other videos, original file playback is stopped? Irritating

6) Back button history is non-intuitive. When you press it, you are redirected to previously played video, instead of the search list where you clicked the video

7) Why when the app is playing in background, it changes your wallpaper?

8) Why displaying `0 thumb ups` when there are no one? Just don't display anything

9) Why comments aren't automatically shown when you load the video?

10) Why the scrolling trackbar for video is red from the left to the right? In the original youtube app as far as I remember, it was red only for the part left to the current position and much lighter for the part of video right to the current position.

11) this app seems to be much slower in comparision with original one, loading new video takes few seconds.

**Comment by opusforlife2** [comment]
> and I am redirected to github.com

That was changed to a newpipe.net link just a couple of days ago. You can try again to confirm this.

> video stops every 10-20 seconds 

Try with the official APK.

The rest of this info can be found in existing issues.

**Comment by SameenAhnaf** [comment]
5, 6. https://github.com/TeamNewPipe/NewPipe/issues/4569

7. Are you using a Motorola device? If so, disabling Moto display might be useful. See: https://www.reddit.com/r/MotoG/comments/ksb7rr/help_how_to_i_disable_motorola_ambient_display/&ved=2ahUKEwir7urd6pGCAxUwsFYBHRLGB6EQFnoECCAQAQ&usg=AOvVaw2Ir38zfqe_oOa161oJm78R

8. https://github.com/TeamNewPipe/NewPipe/issues/5201

9. You can swipe left/right and leave the video on comments' tab. From then on, comments will load at first whenever you open any video.
10. https://github.com/TeamNewPipe/NewPipe/issues/5201
11. Loading subscriptions or search results may take a significant amount of time at first. The time should get reduced once the browser cache is loaded.

**Comment by krzysiobal** [comment]
Thank you for reply!
Ad1) No - it still opens github, instead of starting download (only on mobile phone web browser)
Ad 7) No - I am using Samsung Galaxy S4

**Reply by opusforlife2** [reply]
Could you long-press on the button and see what the URL is?

**Comment by krzysiobal** [comment]
https://github.com/TeamNewPipe/NewPipe/releases

**Reply by opusforlife2** [reply]
How odd. Try refreshing the page? Maybe it's cached in your browser, since it's small in size.

**Comment by opusforlife2** [comment]
Okay, yeah, something is up. @TobiGr I checked the source code of the main page. It shows that the download button has a GH Releases URL. However, when I long press on it, the URL shown is the Newpipe repo. What's going on?

**Reply by opusforlife2** [reply]
Oh wow. The page source shows it as a GH URL. But using Inspect shows the repo URL.

**Comment by krzysiobal** [comment]
I think the GitHub URL is set in HTML code and JavaScript script changes the download url few moments after the page is loaded, because when you entirely disable javascript in your browser, GitHub url stays there.

**Reply by opusforlife2** [reply]
@TheAssassin Maybe the original URL should also be changed to point to the repo instead of GH?
